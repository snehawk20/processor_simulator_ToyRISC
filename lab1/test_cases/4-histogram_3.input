	.data
count:
	0
	0
	0
	0
	0
	0
	0
	0
  0
  0
  0
marks:
	0
  2
  0
  2
  10
  6
  4
  10
  10
  8
  4
  6
  10
  8
  2
  4
  2
  0
  8
  4
n:
  20
