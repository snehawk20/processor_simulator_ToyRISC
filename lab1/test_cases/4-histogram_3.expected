Main Memory Contents:

0		: 3
1		: 0
2		: 4
3		: 0
4		: 4
5		: 0
6		: 2
7		: 0
8		: 3
9		: 0
10		: 4
